uint64 id
geometry_msgs/Point pixel
geometry_msgs/Point location